### PR TITLE
MI-1505: Adding templates to wrap long text - initial config

### PR DESCRIPTION
**Pull request by ericrommel**
Initial development that adds templates to wrap long texts on the views.

### CODE REVIEW

File: cronman/tests/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+@pytest.mark.usefixtures("app1")
+class TestCronTaskAdmin:
+    """ Tests for CronTaskAdmin """
+
+    def test_change_view(self, admin_client):
+        """Test if CronTaskAdmin change view can be loaded."""
+        url = reverse("admin:cronman_crontask_change")
+        response = admin_client.get(url)
+        assert response.status_code == 200
```
Comment (JakeBallantyne) [inline cronman/tests/test_admin.py:19]:
```suggestion
        assert response.status_code == 200
        
        loaded_template = response.templates[0]
        assert (
            loaded_template.name == "cronman/admin/cron_task/change_form.html")
```

Comment (JakeBallantyne) [reply cronman/tests/test_admin.py:19]: to ensure the correct template was loaded

File: cronman/admin.py [hunk 2]
```diff
@@ -31,6 +31,10 @@ class CronTaskAdmin(admin.ModelAdmin):
     list_filter = ("status", "cron_job")
     max_num = 0
 
+    # Custom templates
+    change_form_template = "admin/cronman/cron/cron_task/change_form.html"
+    change_list_template = "admin/cronman/cron/cron_task/change_list.html"
```
Comment (JakeBallantyne) [inline cronman/admin.py:36]:
```suggestion
    change_form_template = "cronman/admin/cron_task/change_form.html"
    change_list_template = "cronman/admin/cron_task/change_list.html"
```

Comment (JakeBallantyne) [reply cronman/admin.py:36]: and change in corresponding admin class, and reorder the templates dir. This is required so this app becomes pluggable into a dependant project. In this case, an overlap would be rare, but if the django admin app included its own `cron_task` template dir, we would be in trouble.

**Review by JakeBallantyne — CHANGES_REQUESTED** [review]
Hey man! Looking good, it works - changes are to clean up the template structure to avoid conflict in the future.

One thing I still don't understand - why are tests skipped with the `setup.py test` command?

File: cronman/tests/test_admin.py (hunk 1 above)
Comment (ericrommel) [reply cronman/tests/test_admin.py:19]: Done

File: cronman/admin.py (hunk 2 above)
Comment (ericrommel) [reply cronman/admin.py]: Done

**Comment by ericrommel** [comment]
> Hey man! Looking good, it works - changes are to clean up the template structure to avoid conflict in the future.
> 
> One thing I still don't understand - why are tests skipped with the `setup.py test` command?

Hey @JakeBallantyne, thank you so much for your help. It is now working. I have cleaned up the code and as we can see, all tests are now passing.

File: cronman/tests/test_admin.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import datetime
+
+from django.urls import reverse
+from django.utils import timezone
+
+from cronman.models import CronTask
+from cronman.tests.base import BaseCronTestCase
+
+
+class TestCronTaskAdmin(BaseCronTestCase):
```
Comment (jbal) [inline cronman/tests/test_admin.py:13]: Can we re-comment `ROOT_URLCONF` and apply this on every test we need the url conf for?
```
@override_settings(ROOT_URLCONF=__name__)
class TestView(TestCase):
        ...
```

**Review by jbal — CHANGES_REQUESTED** [review]
One fast change!

**Review by mbarakaja — APPROVED** [review]
👍🏽

**Review by jbal — APPROVED** [review]
Great stuff man!

**Review by Xyloxy — APPROVED** [review]
LGTM

**Pull request merged**